clients/include/norm.hpp

```diff
@@ -21,28 +28,260 @@
 
 /*! \brief  Template: norm check for general Matrix: half/float/doubel/complex  */
 
-// see check_norm.cpp for template speciliazation
 // use auto as the return type is only allowed in c++14
```

Comment:
We use C++14 for our internal code (not external C interfaces), so it's okay to use `auto` as a placeholder for function return type.

Reply:
understand. update it in commit "update complex change 2".